### PR TITLE
implement solution by React Router

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_people-table-basics/)

### CODE REVIEW

**Review by sergey-lipikhin — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -1,173 +1,59 @@
...
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { Block } from './components/Block/Block';
+import { NavList } from './components/NavList/NavList';
+
+export const App: React.FC = () => {
+  return (
```
Comment (OleksiiBlezniuk) [inline src/App.tsx:13]:
```suggestion
export const App: React.FC = () => (
```

File: src/components/Block/Block.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { PersonList } from '../PersonList/PersonList';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const Block: React.FC = () => {
```
Comment (OleksiiBlezniuk) [inline src/components/Block/Block.tsx:8]: Bad name for component

Comment (OleksiiBlezniuk) [inline src/components/Block/Block.tsx:8]:
```suggestion
export const PeoplePage: React.FC = () => {
```

File: src/components/PersonInfo/PersonInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  isSelected: boolean;
+  father: Person | undefined;
+  mother: Person | undefined;
```
Comment (OleksiiBlezniuk) [inline src/components/PersonInfo/PersonInfo.tsx:10]:
```suggestion
  father?: Person;
  mother?: Person;
```

**Review by OleksiiBlezniuk — APPROVED** [review]

**Pull request closed without merging**